static/admin/config.yml

```diff
@@ -1,6 +1,6 @@
 backend:
   name: git-gateway
-  branch: main
+  branch: careers
```

Comment:
you do you!

As far as I'm concerned, you could just commit to `main` as you're currently the main dev working on this project

Reply:
in this case, faster dev is better dev

Reply:
Switching the branch actually makes things way easier as if I commit the new MD files to main, I then have to pull and rebase main onto my branch to see the changes. Committing straight to the branch lets me pull it immediately without issues. It's a bit annoying to have to do this at all, but it is what it is. 

 It also usually causes the branch to fail since the stuff I'm modifying doesn't exist in the main builds graphQL schema.